### PR TITLE
Uploading of Travis artifacts

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR enables uploading of `Travis` artifacts to `GCS`.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +25,19 @@ after_success:
   - ./scripts/publish-artifacts.sh
 
 script: ./gradlew check --stacktrace
+
+after_script:
+  # Installation of https://github.com/travis-ci/dpl.
+  - gem instal dpl
+  # Reports creation.
```
Comment (armiol) [inline .travis.yml]: `# Prepare the test and coverage reports for the upload.`

File: .travis.yml [hunk 2]
```diff
@@ -23,3 +25,19 @@ after_success:
...
+  - mkdir reports
+  - zip -r reports/test-reports.zip **/build/reports
+  - zip reports/jacoco-reports.zip **/build/jacoco
+  # Reports deployment to GCS.
```
Comment (armiol) [inline .travis.yml]: `#Upload the prepared reports to GCS.`

File: .travis.yml [hunk 3]
```diff
@@ -23,3 +25,19 @@ after_success:
...
 
 script: ./gradlew check --stacktrace
+
+after_script:
+  # Installation of https://github.com/travis-ci/dpl.
+  - gem instal dpl
+  # Reports creation.
+  - mkdir reports
+  - zip -r reports/test-reports.zip **/build/reports
+  - zip reports/jacoco-reports.zip **/build/jacoco
```
Comment (armiol) [inline .travis.yml]: It's weird that you don't have `-r` here.

File: .travis.yml (hunk 1 above)
Comment (dmytro-grankin) [inline .travis.yml]: Done.

File: .travis.yml (hunk 3 above)
Comment (dmytro-grankin) [inline .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (dmytro-grankin) [inline .travis.yml]: Done.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**